scripts/sysman/rasState.yml

```diff
@@ -0,0 +1,87 @@
+version: "1.7"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "RAS Get State Extension Name"
+version: "1.7"
+name: $S_RAS_GET_STATE_EXT_NAME
+value: '"$XS_extension_ras_state"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "RAS Get State Extension Version(s)"
+version: "1.7"
+name: $s_ras_state_ext_version_t
+etors:
+    - name: "1_7"
+      value: "$X_MAKE_VERSION( 1, 7 )"
+      desc: "version 1.7"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension structure for providing RAS error counters for different error sets"
+version: "1.7"
+class: $sRas
+name: $s_ras_state_ext_t
+members:
+    - type: $s_ras_error_cat_t
+      name: "category"
+      desc: "[out] category for which error counter is provided."
+    - type: uint64_t
+      name: errorCounter
+      desc: "[out] Current value of RAS counter for specific error category."
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Ras Get State"
+class: $sRas
+name: RasGetStateExt
+details:
+    - "This function retrieves error counters for different RAS error categories."
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_ras_handle_t
+      name: hRas
+      desc: "[in] Handle for the component."
+    - type: uint32_t*
+      name: pCount
+      desc: |
+            [in,out] pointer to the number of RAS state structures that can be retrieved.
+            if count is zero, then the driver shall update the value with the total number of error categories for which state can be retrieved.
+            if count is greater than the number of RAS states available, then the driver shall update the value with the correct number of RAS states available.
+    - type: $s_ras_state_t*
+      name: pState
+      desc: |
+            [in,out][optional][range(0, *pCount)] array of query results for RAS error states for different categories.
+            if count is less than the number of RAS states available, then driver shall only retrieve that number of RAS states.
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Ras Clear State"
+class: $sRas
+name: RasClearStateExt
+details:
+    - "This function clears error counters for a RAS error category."
+    - "Clearing errors will affect other threads/applications - the counter values will start from zero."
+    - "Clearing errors requires write permissions."
+    - "The application should not call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_ras_handle_t
+      name: hRas
+      desc: "[in] Handle for the component."
+    - type: $x_bool_t
```

Comment:
do we still need the boolean, given the function is called "Clear"?

Reply:
fixed